Dockerfile

```diff
@@ -3,14 +3,14 @@ MAINTAINER colab oratory
 
 # the folder is called /vagrant for compatability with vagrant... should
 # probably be called /oratory
-RUN mkdir -p /vagrant/oratory/ /vagrant/lib
+RUN mkdir -p /vagrant/oratory/ /opt/node/lib/
 
-WORKDIR /vagrant/lib/
-ADD ./oratory/package.json /vagrant/lib/
+WORKDIR /opt/node/lib/
+ADD ./oratory/package.json /opt/node/lib/
 RUN npm install && \
     npm install -g supervisor
-ENV NODE_PATH /vagrant/lib/node_modules:$NODE_PATH
+ENV NODE_PATH /opt/node/lib/node_modules:$NODE_PATH
 
 WORKDIR /vagrant/oratory/
 EXPOSE 80
-
+CMD touch users.js && supervisor server.js
```

Comment:
I think `supervisor` is intended for development and not production. I'm a fan of switching this to `forever`, or if docker can be environment aware then doing either depending.